src/static/postbuild.js

```diff
@@ -3,7 +3,7 @@ import color from "chalk"
 import { writeFile } from "fs-promise"
 import writeAppcache from "../_utils/appcache"
 
-export default (config, files, log) => {
+export default function(config, files, log) {
```

Comment:
if seems that flow can handle arrow function already, what is the error that we got here?